### PR TITLE
Manually adding missing UDFs

### PR DESCRIPTION
**Pull request by aastha25**
Manually registered some  UDF

Testing Done:
Built successfully. Checked spotlessJavaCheck.
Views that failed before can now be successfully translated.

### CODE REVIEW

File: coral-common/src/main/java/com/linkedin/coral/common/functions/FunctionReturnTypes.java [hunk 1]
```diff
@@ -45,6 +45,7 @@ private FunctionReturnTypes() {
   public static final SqlReturnTypeInference BINARY = ReturnTypes.explicit(SqlTypeName.BINARY);
   public static final SqlReturnTypeInference BIGINT = ReturnTypes.explicit(SqlTypeName.BIGINT);
   public static final SqlReturnTypeInference SMALLINT = ReturnTypes.explicit(SqlTypeName.SMALLINT);
+  public static final SqlReturnTypeInference INTEGER = ReturnTypes.explicit(SqlTypeName.INTEGER);
```
Comment (ljfgem) [inline coral-common/src/main/java/com/linkedin/coral/common/functions/FunctionReturnTypes.java:48]: Looks like it's not used?

File: coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java [hunk 2]
```diff
@@ -430,24 +431,35 @@ public boolean isOptional(int i) {
         STRING);
     createAddUserDefinedFunction("com.linkedin.dali.view.udf.entityhandles.PhoneNumberNormalizer",
         FunctionReturnTypes.STRING, STRING_STRING_STRING);
-    createAddUserDefinedFunction("com.linkedin.dwh.udf.sessionization.CleanupBrowserId", FunctionReturnTypes.STRING,
-        STRING);
-    createAddUserDefinedFunction("com.linkedin.dwh.udf.profile.GetProfileUrl", FunctionReturnTypes.STRING, family(
-        SqlTypeFamily.NUMERIC, SqlTypeFamily.STRING, SqlTypeFamily.STRING, SqlTypeFamily.STRING, SqlTypeFamily.STRING));
-    createAddUserDefinedFunction("com.linkedin.dali.views.premium.udf.GetFamily", FunctionReturnTypes.STRING,
-        family(SqlTypeFamily.MAP));
```
Comment (ljfgem) [inline coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java:438]: Is it safe to remove this UDF?

**Review by ljfgem — COMMENTED** [review]
Thanks @aastha25 for this PR!

File: coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java (hunk 2 above)
Comment (aastha25) [reply coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java:438]: Thanks for pointing it out. I added it back

File: coral-common/src/main/java/com/linkedin/coral/common/functions/FunctionReturnTypes.java (hunk 1 above)
Comment (aastha25) [reply coral-common/src/main/java/com/linkedin/coral/common/functions/FunctionReturnTypes.java]: Thanks for pointing it out. I ended up not using it so removing it for now.

**Pull request merged**